blocks/hero/hero.js

```diff
@@ -38,6 +39,11 @@ export default async function decorate(block) {
   let timeString = '';
   if (time && time.content) {
     timeString = time.content;
+    const localisedTime = {
+      en: ' Min Read',
+      de: ' Lesedauer',
+    };
+    timeString += localisedTime[locale] || localisedTime.en;
```

Comment:
I think we should use [placeholders](https://www.hlx.live/docs/placeholders) to automatically take care of the localisation.

Reply:
Nice. I'll check it out.